build/Dockerfile

```diff
@@ -76,24 +76,12 @@ ARG SERVER_PORT=10999
 ENV DESCRIPTION="Powered by DST Academy." \
 	SERVER_PORT="$SERVER_PORT" \
 	MAX_PLAYERS=4 \
-	WHITELIST_SLOTS=0 \
-	PVP_ENABLE=false \
 	GAME_MODE=survival \
```

Comment:
Should we remove this? You made a change in the docs stating that this depends on the `GAME_MODE`, so we should remove this here, no?